fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1039,6 +1040,32 @@ public void testIngestOnSubtree() throws Exception {
 
     }
 
+    /**
+     * @author griffinj
```

Comment:
Method-level @author annotations probably not necessary.
...nor inline ticket reference below.